tests/Api/Request/RequestBuilderTest.php

```diff
@@ -162,6 +170,41 @@ public function testBuild(
         $this->assertEquals($expectedParams, $result->getParams());
     }
 
+    public function getUrlProvider()
+    {
+        return [
+            'Webstore configuration $domainSsl variable is not an empty string' => [
```

Comment:
Use these scenario names instead:
* `Secure domain is configured`
* `Domain is configured`
* `No domain is configured`